src/bulk-load-options.js

```diff
@@ -0,0 +1,15 @@
+// TODO: expand to add KEEP_IDENTITY, USE_INTERNAL_TRANSACTIONS while adding bulk load support for tables
+/**
+ * Enum for BulkLoad options
+ * @readonly
+ */
+module.exports.BULK_LOAD_OPTIONS = {
+  /** Honors constraints during bulk load, it is disabled by default */
+  CHECK_CONSTRAINTS: 'CHECK_CONSTRAINTS',
+  /** Honors insert triggers during bulk load, it is disabled by default */
+  FIRE_TRIGGERS: 'FIRE_TRIGGERS',
+  /** Honors null value passed, ignores the default values set on table */
+  KEEP_NULLS: 'KEEP_NULLS',
```

Comment:
The documentation sais that the option is named 'KEEPNULLS' not "KEEP_NULLS", but why did the test case not fail? Is "KEEP_NULLS" an alias?